File: RK2/master/urls.py
from django.urls import path

from . import views

urlpatterns = [
    path('', views.index),
    path('PC/', views.pcs),
    path('Program/', views.programs),
    path('PC/<int:id>/', views.pc),
    path('Program/<int:id>/', views.program),
]

File: Lab7/src/modules/urls.js
class Urls {
    constructor() {
        this.url = 'http://127.0.0.1:8000/';
    }

    pcs() {
        return `${this.url}PC/`
    }

    pc(id) {
        return `${this.url}PC/${id}/`
    }
}

export const urls = new Urls()